src/Admin/Orders.php

```diff
@@ -226,7 +250,41 @@ private function get_woocommerce_report_data() {
 	 */
 	public function render_recover_carts_modal() {
 
-
+		$abandoned_carts_count = $this->get_abandoned_carts_count();
+		$recovered_revenue     = $this->get_recovered_revenue();
+
+		/* translators: Placeholders: %1$s - <strong> tag, %2$s - abandoned cart estimate, %3$s - </strong> tag, %4$.2f - recovered revenue estimate */
+		$modal_message = sprintf( esc_html__( 'Stores like yours recover an average of %1$s%2$s%3$s carts for %1$s$%4$.2f recovered revenue%3$s per month. Do you want to install Jilt for WooCommerce to start recovering carts? You can then connect to Jilt with one click!', 'sv-wc-jilt-promotions' ),
```

Comment:
To give translators more flexibility it is b better to have each HTML tag use a different placeholder. That way they could, for example, remove the `<strong>` tags for the number of carts, but keep them for the revenue.

Also, to avoid hardcoding the currency symbol in the message, we can use another string placeholder and use `wc_price( $recovered_revenue )` to generate the value. That way we ensure that the amount looks exactly like any other amount WooCommerce shows on the Orders page.

```suggestion
		$modal_message = sprintf( esc_html__( 'Stores like yours recover an average of %1$s%2$s%3$s carts for %4$s%5$s recovered revenue%6$s per month. Do you want to install Jilt for WooCommerce to start recovering carts? You can then connect to Jilt with one click!', 'sv-wc-jilt-promotions' ),
```

Reply:
Changed on https://github.com/skyverge/wc-jilt-promotions/pull/22/commits/cc25a520d9c8adb6f1b9b446f156a3500ddc6e70